### DISCUSSION TITLE
Cannot boot after upgrade to 12.5

### DISCUSSION THREAD

**Discussion by wmf39** [SpiralLinux/SpiralLinux-project#396] [Q&A] [answered]
I work with a VBox VM for testing Spirallinux. After dist-upgrade and autoremove --purge the system no longer boots. The system uses the default encryption setup chosen by installer. When I enter the correct password, it shows the efi-gui. if wrong, it shows the grub-shell. What to do?

**Comment by wmf39** [comment]
I repaired via booting the iso, mounting the crypto-partition, doing some bind-mounts, chroot and doing snapper-rollback. But there is still a problem with the updates, a try with Discover also failed. What am I doing wrong?

**Reply by geckolinux** [reply]
Hi there, we would really need a lot more information about the error messages.

**Comment by AlphaElwedritsch** [comment]
I've updated my stable gnome version today with `sudo apt update && sudo apt list --upgradable && sleep 2 && sudo apt upgrade -y && sudo apt autoremove -y && flatpak update -y` No issues at all

But I didn't do a `dist-upgrade`. I don't know why I should do a `dist-upgrade` either

**Comment by wmf39** [comment, marked as answer]
@geckolinux 
I wished I could, but there are no error messages. The updates run without errors, but on restart it fails with described behaviour. 

@AlphaElwedritsch 
Tested with `apt upgrade` too, but same problem.

Edit: I tested again and rolled back to the first btrfs snapshot generated after fresh install. Updated, worked. So I think, the problem was made when I switched to Debian Testing. Cause I perceived it very unstable, I switched back to stable via btrfs snapshot. Unfortunately I did this with `snapper rollback` and not `snapper-rollback`.  Maybe that was the cause, I have no other explanation.

**Reply by geckolinux** [reply]
Ah, my apologies, I misunderstood that it was throwing error messages when you tried to initiate the update process.

The issue of `snapper rollback` vs `snapper-rollback` sounds like a likely explanation. I would probably recommend backing up your `/home` and re-installing as the easiest solution, if possible.

**Reply by wmf39** [reply]
Thank you, for now I will continue with the updated snapshot-1. It is just for test and there are no important data in `/home`.
